migrate/migrate.go

```diff
@@ -22,6 +22,10 @@ import (
 	"github.com/scylladb/gocqlx/qb"
 )
 
+// AwaitSchemaAgreementBeforeApplyingStatement controls whether to await
+// schema agreement before applying each migration statement.
+var AwaitSchemaAgreementBeforeApplyingStatement = true
```

Comment:
Does this even have value? I'd like to get rid of it, don't know what I was thinking.